### PR TITLE
Add configuration details for APCU

### PR DESCRIPTION
**Pull request by erikhansen**
## Purpose of this pull request

Add configuration details for APCU. For some reason the APCU details added in PR https://github.com/magento/devdocs/pull/2115 were lost at some point, so re-adding them

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/performance-best-practices/software.html
- https://devdocs.magento.com/guides/v2.2/performance-best-practices/deployment-flow.html

whatsnew
Added instructions for configuring the  PHP APCu extension to the "Software recommendations" and "Deployment flow" topics in the _Performance Best Practices Guide_.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/performance-best-practices/deployment-flow.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Run the following command to preprocess and compile DI:
 bin/magento setup:di:compile
 ```
 
-After compilation completes, we recommend running the following command:
+After compilation completes, we recommend running the following command (you should also ensure [APCU is enabled]({{ page.baseurl }}/performance-best-practices/software.html#php-settings)):
```
Comment (meker12) [inline guides/v2.2/performance-best-practices/deployment-flow.md]:
```suggestion
After compilation completes, confirm that [APCU is enabled]({{ page.baseurl }}/performance-best-practices/software.html#php-settings)), then run the following command:
```

File: guides/v2.2/performance-best-practices/software.md [hunk 2]
```diff
@@ -95,6 +95,15 @@ opcache.memory_consumption=64
 opcache.max_accelerated_files=60000
 ```
 
+#### APCU
+
+We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) for maximum performance. This extension caches file locations for opened files, increasing performance for Magento server calls (including pages, ajax calls, and endpoints).
```
Comment (meker12) [inline guides/v2.2/performance-best-practices/software.md]:
```suggestion
We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) to optimize for maximum performance. This extension caches file locations for opened files, increasing performance for Magento server calls (including pages, ajax calls, and endpoints).
```

File: guides/v2.2/performance-best-practices/software.md [hunk 3]
```diff
@@ -95,6 +95,15 @@ opcache.memory_consumption=64
...
+Edit your `apcu.ini` file to include the following:
+
+	extension=apcu.so
+	[apcu]
+	acp.enabled = 1
+
```
Comment (meker12) [inline guides/v2.2/performance-best-practices/software.md]: Use text block formatting for this code example:  
```text
blah, blah, blah
```

File: guides/v2.3/performance-best-practices/software.md [hunk 4]
```diff
@@ -93,6 +93,15 @@ opcache.memory_consumption=64
 opcache.max_accelerated_files=60000
 ```
 
+#### APCU
+
+We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) for maximum performance. This extension caches file locations for opened files, increasing performance for Magento server calls (including pages, ajax calls, and endpoints).
```
Comment (meker12) [inline guides/v2.3/performance-best-practices/software.md]:
```suggestion
We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) to optimize for maximum performance. This extension caches file locations for opened files, increasing performance for Magento server calls (including pages, ajax calls, and endpoints).
```

File: guides/v2.3/performance-best-practices/software.md [hunk 5]
```diff
@@ -93,6 +93,15 @@ opcache.memory_consumption=64
...
+Edit your `apcu.ini` file to include the following:
```
Comment (meker12) [inline guides/v2.3/performance-best-practices/software.md]:
```suggestion

Edit your `apcu.ini` file to include the following:
```

Comment (meker12) [reply guides/v2.3/performance-best-practices/software.md]: Blank line before the instruction Edit your ....

File: guides/v2.2/performance-best-practices/software.md [hunk 6]
```diff
@@ -95,6 +95,15 @@ opcache.memory_consumption=64
 opcache.max_accelerated_files=60000
 ```
 
+#### APCU
+
+We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) for maximum performance. This extension caches file locations for opened files, increasing performance for Magento server calls (including pages, ajax calls, and endpoints).
+Edit your `apcu.ini` file to include the following:
```
Comment (meker12) [inline guides/v2.2/performance-best-practices/software.md]:
```suggestion

Edit your `apcu.ini` file to include the following:
```

Comment (meker12) [reply guides/v2.2/performance-best-practices/software.md]: Blank line before the instruction Edit your ....

File: guides/v2.3/performance-best-practices/software.md [hunk 7]
```diff
@@ -93,6 +93,15 @@ opcache.memory_consumption=64
...
+
+	extension=apcu.so
+	[apcu]
+	acp.enabled = 1
+
```
Comment (meker12) [inline guides/v2.3/performance-best-practices/software.md]: Use text block formatting for this code example:  
```text
blah, blah, blah
```

File: guides/v2.2/performance-best-practices/deployment-flow.md (hunk 1 above)
Comment (meker12) [reply guides/v2.2/performance-best-practices/deployment-flow.md]: Should the reference here be APCu as in other places?

**Comment by dshevtsov** [comment]
https://github.com/magento/devdocs/pull/6076#event-2834023841
@meker12 this PR doesn't contain a new topic.

**Comment by dshevtsov** [comment]
@erikhansen, the DevDocs team moved all the topics into a separate directory `src/` (for more details, see #6077). This is the reason you are getting Conflicting files in this pull request. To resolve this conflict, make sure that the files with your changes were moved to the `src/`directory:

```
src/guides/v2.2/performance-best-practices/deployment-flow.md
src/guides/v2.2/performance-best-practices/software.md
src/guides/v2.3/performance-best-practices/software.md
```

**Comment by meker12** [comment]
@erikhansen Since we haven't heard from you, I created PR [#6192] that fixes the conflicts so that we can get your contribution merged. Thanks for your contribution!

**Pull request closed without merging**